docs/pages/docs/core-concepts/filtering.mdx

```diff
@@ -8,55 +8,67 @@ import HeartIcon from "@heroicons/react/solid/HeartIcon";
 
 # Filtering Workspaces
 
-Turborepo supports a `pnpm`-like `--filter` flag that allows you to select the workspaces
-that will act as "entry points" into your monorepo's workspace/task graph.
-You can filter your project by workspace name, workspace directory, whether workspaces include dependents/dependencies, and by changes
-in git history.
+A monorepo can contain hundreds, or thousands, of workspaces. By default, running `turbo run test` will execute the `test` task in **all available workspaces**.
 
-`turbo` will run each task against each matched workspace, ensuring that any dependencies
-are run first, according to the `pipeline` specification in [`turbo.json`](/docs/reference/configuration#pipeline).
+![Without using a filter, test will be run across all packages](../../../public/images/docs/no-filter.png)
 
-## Filter Syntax
+Turborepo supports a `--filter` flag that lets you **select the workspaces you'd like to execute your task in**.
+
+![With a filter on shared, only the shared package runs test](../../../public/images/docs/with-filter.png)
 
-Filters specify combinations of workspaces, directories, and git commits to act as entrypoints
-for execution.
+You can use it to:
 
-Multiple filters can be combined to select distinct sets of targets. Additionally, filters
-can also exclude targets. A target that matches any filter and is not explicitly excluded will
-be in the final entrypoint selection.
+- Filter by [workspace name](#filter-by-workspace-name)
+- Filter by [workspace directory](#filter-by-directory)
+- Include [dependents](#include-dependents-of-matched-workspaces) and [dependencies](#include-dependencies-of-matched-workspaces) of matched workspaces
+- Execute tasks from the [workspace root](#the-workspace-root)
+- Filter by [changes in git history](#filter-by-changed-workspaces)
+- [Exclude workspaces](#excluding-workspaces) from selection
 
-Turborepo's filter syntax is based on pnpm's filter syntax, so it should feel familiar to pnpm users.
+Turborepo will run each task against each matched workspace, ensuring that any dependent tasks are run first, according to the `pipeline` specification in [`turbo.json`](/docs/reference/configuration#pipeline).
 
-### Filter by workspace
+## Filter Syntax
 
-Supports exact matches (`--filter=my-pkg`), and globs (`--filter=*my-pkg*`) are supported.
-Scoped workspaces (`@foo/bar`) can be abbreviated to just the workspace name (`bar`) as long as the workspace name
-is unique within the monorepo (e.g. `@foo/bar` exists, but `@baz/bar` does not).
+### Multiple filters
 
-The monorepo's root can be selected using the token `//`.
+You can specify multiple filters by adding multiple `--filter` to the command:
```

Comment:
Do we need to specify what happens in this case? Are they both run? We cover this a little below in "excluding workspaces", maybe that's sufficient?

Reply:
Yeah I think it is.